data-sources/github-for-codestream/config.yml

```diff
@@ -0,0 +1,11 @@
+id: github-for-codestream
```

Comment:
shall we skip this one and just use the core `codestream` data source, which is a nerdlet?